std_msgs/Header header
float64[] joint_states

---
uint8 SUCCEEDED=0
uint8 FAILED=1
uint8 result_status